lib/src/hls/better_player_hls_utils.dart

```diff
@@ -137,6 +138,16 @@ class BetterPlayerHlsUtils {
         targetDuration = parsedSubtitle.targetDurationUs! ~/ 1000;
       }
 
+      bool isDefault = false;
+      // bool isForced = false;
+      // bool isAutoSelect = false;
+
+      if(rendition.format.selectionFlags != null) {
+        isDefault = Util.checkBitPositionIsSet(rendition.format.selectionFlags!, 1);
```

Comment:
Why do you need to check that with bits?

Reply:
I just used the pattern that was being used, I try not to change how the parser work.

You can see in file `util.dart`

```
class Util {
  static const int selectionFlagDefault = 1;
  static const int selectionFlagForced = 1 << 1; // 2
  static const int selectionFlagAutoSelect = 1 << 2; // 4
...
}
```
and in the file `hls_playlist_parser.dart`

```
...
if (parseOptionalBooleanAttribute(
        line: line,
        pattern: regexpDefault,
        defaultValue: false)) flags |= Util.selectionFlagDefault;
...
```

these are operation with bits.